src/applications/search/styles.scss

```diff
@@ -90,6 +107,10 @@
           width: unset;
         }
       }
+
+      @media (max-width: $medium-screen - 1) {
+        width: 100%;
+      }
     }
```

Comment:
Our CSS is intended to be mobile-first, so an easy pattern is to start with is mobile style, then apply overrides as the screen sizes increase.  This way we can use the shorthand media queries, like [here](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/sass/modules/_m-veteran-banner.scss#L25), to prevent that `-1` issue.